templates/src/objective-c/BoringSSL-GRPC.podspec.template

```diff
@@ -141,8 +139,8 @@
                         'src/crypto/**/*.{h,c,cc}',
                         # We have to include fiat because spake25519 depends on it
                         'src/third_party/fiat/*.{h,c,cc}',
-                        # Include the err_data.c generated in prepare_command below
-                        'src/err_data.c'
+                        # Include the err_data.c pre-generated in boringssl's master-with-bazel branch
+                        'err_data.c'
```

Comment:
Can you confirm that all the BoringSSL commits that gRPC will use in `/thrird_party/boringssl` are going to be in the `master-with-bazel` branch?

Reply:
Yes, it's kind of required, because this podspec has been adjusted to expect the directory layout of `master-with-bazel` branch.
Also, the sanity tests enforce that third_party/boringssl-with-bazel submodule is at the same commit as this podspec.

Btw, the `third_party/boringssl` submodule no longer exists - it's been removed in my unification PR.

Reply:
Ok thanks!